modules/social_features/social_user_export/src/Plugin/Action/ExportUser.php

```diff
@@ -67,6 +71,27 @@ class ExportUser extends ViewsBulkOperationsActionBase implements ContainerFacto
    */
   protected $config;
 
+  /**
+   * File repository services.
+   *
+   * @var \Drupal\file\FileRepository
+   */
+  protected FileRepository $fileRepository;
```

Comment:
Let's add type for all properties in this class and not only for the new ones.

Reply:
OKay.